lib/page/term_table.rb

```diff
@@ -3,6 +3,35 @@
 require_relative '../everypolitician_extensions'
 require_relative '../person_cards'
 
+# Reopen the LegislativePeriod class to add helper methods; we could
+# consider adding some of these to the class definition in
+# everypolitician-ruby instead.
```

Comment:
Rather than re-opening here, I think it would be better to add these into `lib/everypolitician_extensions.rb`, especially as we're already extending `LegislativePeriod` there.